src/common/Page.ts

```diff
@@ -1587,6 +1587,16 @@ export class Page extends EventEmitter {
     });
   }
 
+  async emulateCPUThrottling(rate: number | null): Promise<void> {
+    assert(
+      rate === null || rate >= 1,
+      'Throttling rate should be greater or equal to 1'
+    );
+    await this._client.send('Emulation.setCPUThrottlingRate', {
```

Comment:
Does it work without the emulation domain enabled?

Reply:
how do we handle errors?

Reply:
The emulation domain does not have `enable`/`disable` commands and the test works. So I think we are good here.
About the error: The [CDP command](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/inspector/inspector_emulation_agent.cc;l=323;bpv=1;bpt=1?q=setCPUThrottlingRate%20-out%2F) only fails if it's called on something else than a page. But since this is added to the `Page` class, we should be good to go. Let me know what you think!